Doppler.MercadoPagoApi/Services/MercadoPagoService.cs

```diff
@@ -0,0 +1,42 @@
+using Doppler.MercadoPagoApi.Models;
+using MercadoPago.Client.Payment;
+using MercadoPago.Resource.CardToken;
+using MercadoPago.Resource.Payment;
+using Microsoft.Extensions.Configuration;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+
+namespace Doppler.MercadoPagoApi.Services
+{
+    public class MercadoPagoService : IMercadoPagoService
+    {
+        private readonly IConfiguration _configuration;
+
+        public MercadoPagoService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public async Task<Payment> CreatePaymentAsync(PaymentCreateRequest paymentCreateRequest)
+        {
+            var mercadoPagoPaymentClient = new PaymentClient();
+
+            var payment = await mercadoPagoPaymentClient.CreateAsync(paymentCreateRequest);
+
+            return payment;
+        }
+
+        public async Task<CardToken> CreateTokenAsync(CardDto card)
+        {
+            using (var client = new HttpClient())
+            {
```

Comment:
You can use a simple using declaration that doesn't require braces:
```suggestion
            using var client = new HttpClient();
```